browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -1358,6 +1408,11 @@ TEST_F(KeyringServiceUnitTest, MigrationPrefs) {
   ASSERT_TRUE(backup_complete);
   EXPECT_TRUE(backup_complete->GetBool());
 
+  const base::Value* selected_account = KeyringService::GetPrefForKeyring(
+      GetPrefs(), kSelectedAccount, mojom::kDefaultKeyringId);
+  ASSERT_TRUE(selected_account);
+  EXPECT_EQ(selected_account->GetString(), "0x111");
+
```

Comment:
also need to test if `kBraveWalletSelectedAccount` is cleared

Reply:
done